gocd/pipelines/symbolicator.yaml

```diff
@@ -42,43 +42,24 @@ pipelines:
                                     ${GO_REVISION_SYMBOLICATOR_REPO} \
                                     sentryio \
                                     "us.gcr.io/sentryio/symbolicator"
-
-            - deploy_canary:
+            # Deploys production and lpq symbolicators.
```

Comment:
Does it still deploy canary too?
Right now we have 3 configs in freight:
- `canary` deploys just that
- `lpq` deploys just that
- `production` deploys all of them

That way, when we deploy to production we know everything is running the deployed version.

Reply:
I'm confused - you said you wanted to leave long running experiments in canary, so I took that as prod should not deploy canary.

Do you want this?
- separate canary deployment pipeline
- separate lpq deployment pipeline
- prod deployment pipeline that covers everything (as it was before)

Reply:
yes, canary is always an intermediate step, and our goal in this quarter is to adapt our processes to deploy master to canary automatically.